src/ImageSharp/IO/ChunkedMemoryStream.cs

```diff
@@ -547,6 +543,16 @@ private void ReleaseMemoryChunks(MemoryChunk chunk)
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static int GetChunkSize(int i)
+        {
+#pragma warning disable IDE1006 // Naming Styles
+            const int _128K = 1 << 17;
```

Comment:
A message to our future selves.
```suggestion
            // Increment chunks sizes with moderate speed, but without using too many buffers from the same ArrayPool bucket of the default MemoryAllocator.
            //https://github.com/SixLabors/ImageSharp/pull/2006#issuecomment-1066244720
#pragma warning disable IDE1006 // Naming Styles
            const int _128K = 1 << 17;
```